src/java-opentimelineio/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+cmake_minimum_required(VERSION 3.13 FATAL_ERROR)
+
+project(jotio VERSION 0.1.0 LANGUAGES CXX)
+
+set(CMAKE_CXX_EXTENSIONS OFF)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
@KarthikRIyer - spoke to @reinecke and 11 is the correct version. Hoping you can just update this directly on your branch.

Reply:
Yes @margant, I had that in mind but forgot to change it today. I'll update this tomorrow.